.pre-commit-config.yaml

```diff
@@ -3,3 +3,5 @@ repos:
     rev: v8.16.1
     hooks:
       - id: gitleaks
+
+default_install_hook_types: [pre-commit, gitleaks]
```

Comment:
Looking into this further, I think I misunderstood it before. The gitleaks hook is already a hook type of "pre-commit", and since that's already the default, we can drop this definition of `default_install_hook_types`.